source-gcs/main.go

```diff
@@ -169,6 +171,20 @@ func main() {
 				"title":       "Prefix",
 				"description": "Prefix within the bucket to capture from"
 			},
+			"advanced": {
+				"properties": {
+				  "ascendingKeys": {
+					"type":        "boolean",
+					"title":       "Ascending Keys",
+					"description": "Improve sync speeds by listing files from the end of the last sync, rather than listing the entire bucket prefix. This requires that you write objects in ascending lexicographic order, such as an RFC-3339 timestamp, so that key ordering matches modification time ordering.",
```

Comment:
This description is really long, and yet it still doesn't communicate everything I'd like it to. Maybe we should add a "For more information see go.estuary.dev/ascending-keys" short link or something?

(For instance I'd like to include a cautionary note about only using this if you're responsible for generating the data and can guarantee that files will always be uploaded strictly in order, and give an explicit example like "If you upload files 'X', 'Z', and 'Y' in that order, it is likely that 'Y' will never be captured).